src/components/email/feedback-requested.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  Html,
+  Body,
+  Container,
+  Heading,
+  Text,
+  Button,
+  Link,
+  Tailwind,
+} from "@react-email/components";
```

Comment:
These special React components render well among various e-mail providers (e.g. Gmail, Outlook, etc.) so it is best to use them and not to import application components.